### DISCUSSION TITLE
How to load page from local storage?

### DISCUSSION THREAD

**Discussion by RMichaelPickering** [Voltstro-Studios/UnityWebBrowser#288] [General]
I'm actually not sure whether 'local' is used in the sense of, local WITHIN THE UNITY PROJECT (perhaps in resources folder?) or local on the current machine's attached storage, which on Windows would look something like "c:\webcontent\index.html". So how do we use local storage on Windows? And do URLs to files on local storage need to specify absolute path or only relative path?

**Comment by PreciousNyasulu** [comment]
what if you serve the pages through a server on your local machine

**Comment by Voltstro** [comment]
Really two ways of currently loading files locally. You can use Chromium's default `file://` protocol to load files from the disk, like `file:///C:/test.html`.

There is also [`LoadHtml`](https://projects.voltstro.dev/UnityWebBrowser/latest/api/voltstrostudios.unitywebbrowser.core.webbrowserclient/#VoltstroStudios_UnityWebBrowser_Core_WebBrowserClient_LoadHtml_System_String_) method. That allows loading HTML content locally. Still, you can't use locally linked resources (JS, CSS, ect). You will either have to embed them into the HTML content, or link them from an external site.
